lib/commands/run.ts

```diff
@@ -32,9 +32,9 @@ export class RunCommandBase implements ICommand {
 			this.platform = this.$devicePlatformsConstants.Android;
 		}
 
-		const availablePlatforms = this.platform ? [this.platform] : this.$platformsData.availablePlatforms;
-		for (let platform of availablePlatforms) {
-			const platformData = this.$platformsData.getPlatformData(platform, this.$projectData);
+		const availablePlatforms = this.platform ? {[this.platform]: this.platform} : this.$platformsData.availablePlatforms;
+		for (var key in availablePlatforms) {
```

Comment:
We like to stick with the ES2015 scoped variables keywords.
Semantically it is more correct to use `const` here instead of `var` or `let`.